public/locales/de.json

```diff
@@ -178,7 +181,8 @@
   },
   "pricer": {
     "rentalPeriodLabel": "Mietdauer",
-    "totalPriceLabel": "Gesamtpreis"
```

Comment:
Can you change `"rentalPeriodLabel": "Mietdauer"` above  to `"registrationPeriodLabel": "Registration Period"` ? That part causes conflicts when I try to merge.